### ISSUE TITLE
SIPssert - Running new configuration

### ISSUE THREAD

**Issue by eliakimfelmorais** [OpenSIPS/SIPssert#19]
I started using Sipssert and I have a few questions:

- Do I need to install sipp if I have some tests using sipp uac ?
- To run sipssert I tried to use this command:

```
[root@ssw-6-x-dev dialplan]# sipssert -t local/01.local-pstn 01.local-pstn
======= Running SIPssert Testing Framework =======
Summary: Total     Passed    Failed
         0         0         0

```
but I got this output like neither test was found. Here follow my files https://github.com/eliakimfelmorais/sipssert-tests if you want to look

Thank you.

**Comment by razvancrainea** [comment]
@eliakimfelmorais your command should be ` sipssert -t local/01.local-pstn .`

**Comment by eliakimfelmorais** [comment]
I tried but It didn't worked 

It seems the sipssert found the scenario but not execute them.

```
[root@ssw-6-x-dev dialplan]# sipssert -t local/01.local-pstn .
======= Running SIPssert Testing Framework =======
=========== Running test set: dialplan ===========
Summary: Total     Passed    Failed
         0         0         0

```

**Comment by razvancrainea** [comment]
Oh, you are right, the command should be:

**Comment by eliakimfelmorais** [comment]
Hi Razvan, 

I tried but this problem happened:
```
[root@ssw-6-x-dev dialplan]# sipssert -t local/01.local-pstn local
======= Running SIPssert Testing Framework =======
Traceback (most recent call last):
  File "/bin/sipssert", line 4, in <module>
    __import__('pkg_resources').run_script('sipssert==0.1.0', 'sipssert')
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 654, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1434, in run_script
    exec(code, namespace, namespace)
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/EGG-INFO/scripts/sipssert", line 9, in <module>
    run_test_sipssert()
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/EGG-INFO/scripts/sipssert", line 6, in run_test_sipssert
    main.main()
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/sipssert/main.py", line 78, in main
    ctrl.run()
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/sipssert/controller.py", line 75, in run
    test_set_obj = tests_set.TestsSet(test_set, self, self.filters)
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/sipssert/tests_set.py", line 46, in __init__
    self.config = config.Config(self.set_path, CONFIG, VARIABLES, controller.config.get_defines())
  File "/root/.local/lib/python3.6/site-packages/sipssert-0.1.0-py3.6.egg/sipssert/config.py", line 74, in __init__
    self.parse(self.template_file_path, template_vars)
TypeError: unsupported operand type(s) for |: 'dict' and 'dict'

```

With the command 

```
sipssert -t local/01.local-pstn . 
```

or 
```
[root@ssw-6-x-dev dialplan]# sipssert -t local/01.local-pstn 01.local-pstn
======= Running SIPssert Testing Framework =======
Summary: Total     Passed    Failed
         0         0         0
```

The sipssert found the scenario but not run.

Is there something wrong in the installation ?

**Comment by razvancrainea** [comment]
@eliakimfelmorais the correct command is `sipssert -t local/01.local-pstn local`, however you are using an old python (3.6) which does not support the dictionary union `|` operator. This is only available starting with python 3.9

Can you try the attached patch? It replaces the pipe operator with a manual union.
[sipssert-dict-union.txt](https://github.com/OpenSIPS/SIPssert/files/13493153/sipssert-dict-union.txt)

**Comment by eliakimfelmorais** [comment]
Sorry Razvan, my bad.

I forgot the python3.9 stuff, by the way I'll use a 3.9 version.

Thank you!
